docs/TAP_SUBMISSION_REQUIREMENTS.md

```diff
@@ -0,0 +1,161 @@
+# TAP SUBMISSION REQUIREMENTS
+This is the doc that will outline all requirements for tap contributors.
+
+Before working on your project, please check that someone else has not submitted the same project already.
+The [Singer website][singer-io] and the [Singer Slack][singer-slack] are great resources for this.
+
+
+Before submitting your tap for review, make sure that you have completed all of the following:
+
+- [Project Structure](#projectstructure)
+- [Catalog](#catalog)
+- [Stream selection](#stream-selection)
+- [Field selection](#field-selection)
+- [Child Streams Handled Correctly](#how-to-handle-child-streams)
+- [Pylint](#pylint)
+- [PyPi](#pypi)
+- [Contributor License Agreement](#contributor-license-agreement)
+
+## Project Structure
+
+The project should have the following structure:
+
+```
+tap-<tap_name>
+├── LICENSE
+├── MANIFEST.in
+├── README.md
+├── setup.py
+├── tap_<tap_name>
+│   ├── schemas
+│   │   ├── <stream_name>.json
+│   │   ├── <stream_name>.json
+│   │   └── <stream_name>.json
+│   ├── client.py
+│   ├── discover.py
+│   ├── __init__.py
+│   ├── streams.py
+│   └── sync.py
+└── tests
+    └── base.py
```

Comment:
Can we add a test example here instead of just base.py? Like test_bookmarks.py or whatever?